Added Kafka Bridge tracing documentation (#2123)
strimzi/strimzi-kafka-operator: documentation/assemblies/assembly-setting-up-tracing-mirror-maker-connect-bridge.adoc

// This assembly is included in the following assemblies:
//
// assembly-tracing.adoc
[id='assembly-setting-up-tracing-mirror-maker-connect-bridge-{context}']
= Setting up tracing for Mirror Maker, Kafka Connect, and the Kafka Bridge

Distributed tracing is supported for Mirror Maker, Kafka Connect (including Kafka Connect with Source2Image support), and the {ProductName} Kafka Bridge.

.Tracing in Mirror Maker

For Mirror Maker, messages are traced from the source cluster to the target cluster; the trace data records messages entering and leaving the Mirror Maker component.

.Tracing in Kafka Connect

Only messages produced and consumed by Kafka Connect itself are traced. To trace messages sent between Kafka Connect and external systems, you must configure tracing in the connectors for those systems. For more information, see xref:assembly-deployment-configuration-kafka-connect-{context}[].

> .Tracing in the the Kafka Bridge

Reviewer: Repeated word: the

Messages produced and consumed by the Kafka Bridge are traced. Incoming HTTP requests from client applications to send and receive messages through the Kafka Bridge are also traced.

> In order to have and end-to-end tracing, you must configure tracing in your HTTP clients.

Reviewer: ```suggestion
In order to have end-to-end tracing, you must configure tracing in your HTTP clients.
```

// This assembly is included in the following assemblies:
//
// assembly-tracing.adoc
[id='assembly-setting-up-tracing-mirror-maker-connect-bridge-{context}']
= Setting up tracing for Mirror Maker, Kafka Connect and Kafka Bridge

> Distributed tracing is supported for Mirror Maker, Kafka Connect (including Kafka Connect with Source2Image support) and Kafka Bridge.

Author: @laidan6000 why should we put `{ProductName}` here?

For Mirror Maker, messages are traced from the source cluster to the target cluster; the trace data records messages entering and leaving the Mirror Maker component.

Only messages produced and consumed by Kafka Connect itself are traced. To trace messages sent between Kafka Connect and external systems, you must configure tracing in the connectors for those systems. For more information, see xref:assembly-deployment-configuration-kafka-connect-{context}[].

> 

Reviewer: Add a third sub-heading:

.Tracing in the the Kafka Bridge

For Kafka Bridge, messages produced and consumed are traced as well as the HTTP requests, coming from clients, about sending and receiving messages through the bridge itself.

> In order to have and end to end tracing, you must configure tracing in your HTTP clients.

Reviewer 2: ```suggestion
In order to have and end-to-end tracing, you must configure tracing in your HTTP clients.
```

> In order to have and end to end tracing, you must configure tracing in your HTTP clients.

Reviewer: ```suggestion
In order to achieve end-to-end tracing of messages, you need to configure distributed tracing in your HTTP clients.
```
